web/src/components/vis/Heatmap.vue

```diff
@@ -705,6 +761,19 @@ export default {
   justify-content: center;
   align-content: center;
 }
+.gridLarge {
+  position: relative;
+  top: 4px;
+  left: 4px;
+  right: 8px;
+  bottom: 8px;
+  display: grid;
+  grid-template-areas: "legend column dl"
+    "row matrix rlabel"
+    "rc clabel ll";
+  justify-content: left;
+  align-content: left;
+}
```

Comment:
Refactor common styles into a common class, switch to scss.

```scss
.grid {
   ... common styles;
   .grid--large {
      ...large styles that are different;
    }
}
```

Reply:
@subdavis https://github.com/girder/viime/pull/609/commits/793ef7b50708c4dec35616ac1a5fbe25af82cc8d